### DISCUSSION TITLE
Regarding licensing / future pricing

### DISCUSSION THREAD

**Discussion by deajan** [rustdesk/rustdesk#1440] [Q&A] [answered]
Hello,

Just found out this project after searching for a couple of years for a good selfhosted Teamviewer alternative.
Looks really good, bandwidth usage is absolutely great, and UI feels snappy.

Browsing the source, I noticed files `https://github.com/rustdesk/rustdesk/blob/master/src/license.rs` and also a file named `lic_main.rs` in earlier versions.

Is a paid version of rustdesk planned ? Or is a community/enterprise fork planned ?
Not that I can't live with it, it's just that I've seen too much good remote desktop tools going crazy in terms of pricing (like screenconnect which had a 300$ license and suddently the same license became 2000$ a couple of years ago, then teamviewer that had one-shot licenses before now only doing monthly subscriptions).
So if a paid version will be made, I'd love to know what's the pricing before joining ;)

Same question for rustdesk-server... Any paid version planned ? If so, what are the plans ?

To be more clear, as of today, I use a solution which I pay as one shot, and can use forever, including 1 year updates.
If I need a newer version, I can get another year updates for 25% of the initial price.
The solution isn't just that good overall, buggy and quite slow.
I can live with such a pricing schema, as long as there's a promise to never skyrocket prices or going subscription only mode.

Best regards.

PS: Sorry if above questions sound strange, it just sounds too good to be true to finally have a real open source solution that can replace Teamviewer without any phony business schemes

**Comment by ganzgustav22** [comment]
Anybody got a comment about the above?

**Comment by deajan** [comment]
Hello @rustdesk ?
I truly understand that you're busy. Could you just take 5 minutes to make a statement like "we're not going to pull an evil subscription scheme later" ? Thanks ;)

**Comment by deajan** [comment]
Hello @rustdesk. Anyone there ?

**Reply by ganzgustav22** [reply]
They're busy implementing licensing schemes :grin:

**Reply by deajan** [reply]
That's honestly not a problem, as long as the licensing is not shady.
I am willing to pay for a good program, as long as the licensing respects good standards, meaning:

- Software should not stop working after license is outdated, indeed no more updates are allowed
- License should guarantee that self hosting option will **always** exist
- License should guarantee that price increases cannot be more than a given percentage annually
- License should guarantee that renewing the license should never cost more than a given percentage of the initial license value
- Retrocompatibility should be kept as much as possible (that's the shady thing Teamv**** does... every year new version that brings nothing but incompatibility, meaning you have to renew whether you want or not)

I have no problem being a paying customer if the solution is good.
I've used way too much bad remote desktop software to be spoiled and not wanting to pay a cent for a decent software (pun intended).

**Comment by Nopalin** [comment]
@deajan  As far as i know, this project first was private, then they changed it to be opensource but not as is, they are progressively migrating functions, so until know you could find these files because are inherit from the private version, but i suppose the will be removed. For this same reason other functions does not work like accounting.

**Comment by deajan** [comment]
@Nopalin Thanks for your insight ;)
I still think there could be an official statement so people don't get afraid of the product.

**Comment by test2a** [comment]
https://github.com/rustdesk/rustdesk/issues/4727

i opened an issue where login button was not working. not that i needed it anyway but i figured it must be some bug but the issue was closed with weird "rustdesk pro" and closed for contributors only. what does that mean?

anyone have any idea @deajan ?

or @rustdesk can you please clarify what do you mean by pro? we are not against you blocking some features for future paid version but lack of clarity in the matter is really not helping. do we know what pro will have over free? will selfhosted be pro also or only free so pro will be closed source?

**Reply by deajan** [reply]
Looks like using unattended remote computers linked to an account are pro / paid features.

**Comment by rustdesk** [comment, marked as answer]
https://rustdesk.com/pricing.html
